src/main/java/uk/gov/ch/controller/update/trusts/TrustLinksController.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.ch.controller.update.trusts;
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.ch.OracleQueryApplication;
+import uk.gov.ch.exception.TrustDataCountNotFoundException;
+import uk.gov.ch.model.update.trusts.TrustLinkData;
+import uk.gov.ch.service.update.trusts.TrustLinksService;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+import uk.gov.companieshouse.logging.util.DataMap;
+
+@RestController
+public class TrustLinksController {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(
+            OracleQueryApplication.APPLICATION_NAME_SPACE);
+    @Autowired
+    private TrustLinksService individualTrusteesService;
+
+    @GetMapping("/overseas-entity/{oeNumber}/trusts/links")
```

Comment:
Shoudl this be 
/overseas-entity/{oeNumber}/trusts/beneficial_owners/links

Reply:
I'll change that now though it does say "trusts/links" on the ticket 

![image](https://github.com/companieshouse/oracle-query-api/assets/124157110/dd0cad49-12f3-4dd7-b209-95567c5b7ea1)